### PR TITLE
fix(module: affix): affix to the viewport (#1327)

### PR DESCRIPTION
**Pull request by skystardust**
* fix target to element

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/ant-design-blazor/ant-design-blazor?pullRequest=1335) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/ant-design-blazor/ant-design-blazor?pullRequest=1335) before we can accept your contribution.<br/><br/>:x: skystardust [sign now](https://cla.dotnetfoundation.org/ant-design-blazor/ant-design-blazor?pullRequest=1335)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/ant-design-blazor/ant-design-blazor?pullRequest=1335) it.</sub>

**Comment by ElderJames** [comment]
Thanks for contribution. Please sign the CLA first.

File: components/affix/Affix.razor.cs [hunk 1]
```diff
@@ -58,6 +58,9 @@ private bool Affixed
         [Parameter]
         public ElementReference Target { get; set; }
 
+        [Parameter]
+        public string TargetId { get; set; }
```
Comment (ElderJames) [inline components/affix/Affix.razor.cs:62]: Please use css selecter, don't only use id.

**Review by ElderJames — CHANGES_REQUESTED** [review]

**Comment by ElderJames** [comment]
Hi @skystardust , please sign the CLA.

**Pull request merged**